test/e2e/dra/test-driver/app/server.go

```diff
@@ -211,9 +212,12 @@ func NewCommand() *cobra.Command {
 				return fmt.Errorf("parse resource config %q: %w", *resourceConfig, err)
 			}
 		}
+		if resources.DriverName == "" {
```

Comment:
You mentioned that the value in a config file overrides the value specified on the CLI. Is this common in these tests? I know that for CLIs that we build internally using urfave, the usual order of [precedence](https://cli.urfave.org/v2/examples/flags/#precedence) is cli > envvar > config > default. Is this different with `cobra`?

Reply:
There's no established order. In this case not having the flag override the config was convenient because then I don't need to change the default (which is currently non-empty).

But you are right, flag overriding config is more common, and by checking whether the flag was set vs. having the default value I can achieve that. Will fix.

Reply:
Done.